mopidy/core/history.py

```diff
@@ -57,3 +57,21 @@ def get_history(self):
         :rtype: list of (timestamp, :class:`mopidy.models.Ref`) tuples
         """
         return copy.copy(self._history)
+
+    def _export_state(self):
+        """Internal method for :class:`mopidy.Core`."""
+        history_list = []
+        for timestamp, track in self._history:
+            history_list.append(models.HistoryTrack(
+                                timestamp=timestamp, track=track))
+        return models.HistoryState(history=history_list)
+
+    def _restore_state(self, state, coverage):
+        """Internal method for :class:`mopidy.Core`."""
+        if state:
+            if not isinstance(state, models.HistoryState):
```

Comment:
Use `internal.check_instance` for consistent error messages / types with the rest of core, or just remove if we switch to an overall state model as per an earlier comment.